BackofficeBundle/Resources/public/coffee/UserFormView.coffee

```diff
@@ -0,0 +1,26 @@
+###*
+ * @namespace OpenOrchestra
+###
+window.OpenOrchestra or= {}
```

Comment:
can you add this in namspace in window.OpenOrchestra.User
and move it in the folder user ?